freqtrade/exchange/exchange.py

```diff
@@ -181,24 +196,34 @@ def set_sandbox(self, api, exchange_config: dict, name: str):
                                      "Please check your config.json")
                 raise OperationalException(f'Exchange {name} does not provide a sandbox api')
 
-    def _load_async_markets(self) -> None:
+    def _load_async_markets(self, reload=False) -> None:
         try:
             if self._api_async:
-                asyncio.get_event_loop().run_until_complete(self._api_async.load_markets())
+                asyncio.get_event_loop().run_until_complete(
+                    self._api_async.load_markets(reload=reload))
 
         except ccxt.BaseError as e:
             logger.warning('Could not load async markets. Reason: %s', e)
             return
 
-    def _load_markets(self) -> Dict[str, Any]:
+    def _load_markets(self) -> None:
         """ Initialize markets both sync and async """
         try:
-            markets = self._api.load_markets()
+            self._api.load_markets()
             self._load_async_markets()
-            return markets
+            self._last_markets_refresh = arrow.utcnow().timestamp
         except ccxt.BaseError as e:
             logger.warning('Unable to initialize markets. Reason: %s', e)
-        return {}
+
+    def _reload_markets(self) -> None:
+        """Reload markets both sync and async, if refresh interval has passed"""
+        # Check whether markets have to be reloaded
+        if (self._last_markets_refresh > 0) and (
+                self._last_markets_refresh + self.markets_refresh_interval
+                > arrow.utcnow().timestamp):
+            return None
+        logger.debug("Performing scheduled market reload..")
```

Comment:
I think you'll want to "reset" `_last_markets_refresh` here too - or call `_load_markets` with a parameter skipping the async load.

Currently it'll skip refresh for the first hour - and then refresh on every iteration ... since `_last_markets_refresh` is not updated anywhere ...

Reply:
Totally right, thanks for the thorough review! Will get that fixed.